basket/news/tasks.py

```diff
@@ -741,10 +741,29 @@ def process_subhub_event_customer_created(data):
         'payee_id': data['customer_id']
     }
 
-    user_data = get_user_data(email=data['email'])
+    user_data = None
+    # try getting user data with the fxa_id first
+    user_data_fxa = get_user_data(fxa_id=contact_data['fxa_id'],
+                                  extra_fields=['id'])
```

Comment:
nit: is this `id` from `extra_fields` actually used? same for line 763.

Reply:
I've been thinking about what to do with that. It's annoying to have to include that in every call. It's there because I need it to return the `id` field so that when I pass the `user_data` dict to `sfdc.update()` it'll have the ID in there to use to update the record. `id` is excluded normally so that we're sure to avoid sending it out with the other user data in HTTP responses, but it can be filtered at that point.